catalog/resource_online_table.go

```diff
@@ -59,6 +59,7 @@ func ResourceOnlineTable() common.Resource {
 			common.CustomizeSchemaPath(m, "spec", "source_table_full_name").SetCustomSuppressDiff(common.EqualFoldDiffSuppress)
 			common.CustomizeSchemaPath(m, "name").SetRequired().SetForceNew()
 			common.CustomizeSchemaPath(m, "status").SetReadOnly()
+			common.CustomizeSchemaPath(m, "unity_catalog_provisioning_state").SetComputed()
```

Comment:
it's better to use `.SetReadOnly()` because it will set optional and required into correct states

Reply:
we also need to add it to the docs?

Reply:
Thanks Alex. Could you explain when it makes sense to use computed properties versus readonly properties?

Reply:
The `SetReadOnly` function does a few things:

* set `Computed` - same as `SetComputed`
* It sets `Required` to `false` and `Optional` to `true` because `Computed` can't be used with `Required`.
* Sets `MinItems` to 0...

In general, if the field is already `Optional`, both functions do the same, but `SetReadOnly` is just safer here because you don't need to check if the field already marked as optional or not.